app/frontend/datatables/tablePipelineInboxColumnSearch.js

```diff
@@ -0,0 +1,40 @@
+import $ from "jquery";
+import { cssSelectors, defaults } from "./config";
+
+$(document).ready(function () {
+  $(cssSelectors.PipelineInboxConfig).DataTable({
+    ...defaults,
+    initComplete: function () {
```

Comment:
There is a lot going on here. Maybe worth adding some documentation? or even splitting it up.

Reply:
Ok, doing a bit of refactor there